SS2-Project/Assets/Starstorm2/Modules/EntityStates/AcidBug/HurtState.cs

```diff
@@ -0,0 +1,17 @@
+﻿using UnityEngine;
+namespace EntityStates.AcidBug
+{
+    public class HurtState : HurtStateFlyer
+    {
+
+        public override void OnEnter()
```

Comment:
I guess since this is our first flying enemy I dont quite understand why we need this hurt state?